docs/docs/hydroflow/ecosystem.md

```diff
@@ -4,24 +4,22 @@ sidebar_position: 4
 
 # The Hydro Ecosystem
 The Hydro Project is an evolving stack of libraries and languages for distributed programming.
-A rough picture of the envisioned Hydro stack is below:
+A rough picture of the Hydro stack is below:
 
 ![Hydro Stack](./img/hydro_stack.png)
 
-The core of the Hydro stack is shown in in the grey box; components that have not been implemented are in orange. 
+Working down from the top:
 
-Working up from the bottom:
+- [*Hydroflow+*](../hydroflow_plus) is an end-user-facing high-level [choreographic](https://en.wikipedia.org/wiki/Choreographic_programming) [dataflow](https://en.wikipedia.org/wiki/Dataflow_programming) language. Hydroflow+ is a *global* language for programming a (potentially elastic) fleet of processes. It is insensitive to the specific deployment of the code—the partitioning of functionality and data across processes, the number of replicas of each process, etc. Instead, programmers provide specifications . The Hydrolysis compiler then generates Hydroflow processes that can be deployed by HydroDeploy to meet those specifications.
```

Comment:
HF+ isn't insensitive to partitioning computing across locations, in fact it's explicit in the code where that takes place (and even our optimizations rely on that information in some places).

Reply:
Revised. @shadaj please review.